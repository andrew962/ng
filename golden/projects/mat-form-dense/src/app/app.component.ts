import { Component } from '@angular/core';
import { FormControl, Validators } from '@angular/forms';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styles: []
})
export class AppComponent {
  title = 'mat-form-dense';
  test = new FormControl('', [Validators.required, Validators.maxLength(2)]);
}
