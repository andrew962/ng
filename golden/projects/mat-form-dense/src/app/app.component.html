<div class="container">
  <div class="row py-4">
    <div class="col-6">
      <mat-form-field class="w-100 dense">
        <mat-label>Nombre</mat-label>
        <input matInput required>
        <mat-icon matSuffix>sentiment_very_satisfied</mat-icon>
        <!-- <mat-error *ngIf="test.invalid">Error</mat-error> -->
        <mat-hint>Hint</mat-hint>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100">
        <mat-label>Test</mat-label>
        <input matInput placeholder="label">
        <mat-icon matPrefix>sentiment_very_satisfied</mat-icon>
        <mat-hint>Hint</mat-hint>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="dense w-100">
        <mat-label>Choose a date </mat-label>
        <input matInput [matDatepicker]="picker">
        <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
        <mat-datepicker #picker></mat-datepicker>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100">
        <mat-label>Date</mat-label>
        <input matInput [matDatepicker]="picker2">
        <mat-datepicker-toggle matSuffix [for]="picker2"></mat-datepicker-toggle>
        <mat-datepicker #picker2></mat-datepicker>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="dense w-100">
        <mat-label>Select</mat-label>
        <mat-select>
          <mat-option value="option">Option</mat-option>
        </mat-select>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100">
        <mat-label>Select</mat-label>
        <mat-select>
          <mat-option value="option">Option</mat-option>
        </mat-select>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100 dense">
        <mat-label>Textarea</mat-label>
        <textarea matInput></textarea>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100">
        <mat-label>Textarea</mat-label>
        <textarea matInput></textarea>
      </mat-form-field>
    </div>
    <!-- <div class="col-6">
      <mat-form-field class="w-100 dense">
        <mat-label>Amount</mat-label>
        <input matInput type="number" class="text-right">
        <span matPrefix>$&nbsp;</span>
        <span matSuffix>.00</span>
      </mat-form-field>
    </div>
    <div class="col-6">
      <mat-form-field class="w-100">
        <mat-label>Amount</mat-label>
        <input matInput type="number" class="">
        <span matPrefix>$&nbsp;</span>
        <span matSuffix>.00</span>
      </mat-form-field>
    </div> -->
  </div>
</div>